Servers/WindowServer/Menu.cpp

```diff
@@ -399,12 +399,12 @@ void Menu::event(Core::Event& event)
             do {
                 if (m_hovered_item_index == 0)
                     m_hovered_item_index = m_items.size() - 1;
-                else if (m_hovered_item_index < 0)
-                    return;
```

Comment:
This is changing logic while the commit message makes it sound like it's only adding assertions.

Reply:
Ah! Silly me - fair point. Thanks :)